src/components/flowchart/draw.js

```diff
@@ -174,6 +174,7 @@ export const drawNodes = function (changed) {
     enterNodes
       .append('rect')
       .attr('class', 'pipeline-node__parameter-indicator')
+      .on('mouseover', this.handleParamsIndicatorMouseOver)
```

Comment:
Optional: in the event that the newparams flag still exists, we would need to cater for the `newparams` flag here as well